File: eSalon/eSalon.Model/Arhiva.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eSalon.Model
{
    public class Arhiva
    {
        public int ArhivaId { get; set; }

        public int KorisnikId { get; set; }

        public int UslugaId { get; set; }

        public DateTime DatumDodavanja { get; set; }

        public string? UslugaNaziv { get; set; }

        public decimal? Cijena { get; set; }

        public byte[]? Slika { get; set; }

        public int? Trajanje { get; set; }
    }
}

File: eSalon/UI/esalon_desktop/lib/models/arhiva.dart
import 'package:json_annotation/json_annotation.dart';

part 'arhiva.g.dart';

@JsonSerializable()
class Arhiva {
  int? arhivaId;
  int? korisnikId;
  int? uslugaId;
  DateTime? datumDodavanja;
  String? uslugaNaziv;
  double? cijena;
  String? slika;
  int? trajanje;

  Arhiva({
    this.arhivaId,
    this.korisnikId,
    this.uslugaId,
    this.datumDodavanja,
    this.uslugaNaziv,
    this.cijena,
    this.slika,
    this.trajanje,
  });

  factory Arhiva.fromJson(Map<String, dynamic> json) => _$ArhivaFromJson(json);
  Map<String, dynamic> toJson() => _$ArhivaToJson(this);
}
